README.md

```diff
@@ -15,7 +15,10 @@ The terraform script builds out a new VPC in your account and 3 corresponding su
 1. Clone the repo: `git clone https://github.com/upmc-enterprises/kubeadm-aws.git`
 - [Install Terraform](https://www.terraform.io/intro/getting-started/install.html)
 - Generate token: `python -c 'import random; print "%0x.%0x" % (random.SystemRandom().getrandbits(3*8), random.SystemRandom().getrandbits(8*8))'`
-- Build out infrastructure: `terraform apply -var 'k8stoken=<token>' -var 'access_key=<key>' -var 'secret_key=<secret>' -var 'key_name=keypair'`
+- Generate ssh keys: `ssh-keygen -f k8s-test`
+- Run terraform plan: `terraform plan -var k8s-ssh-key="$(cat k8s-test.pub)" -var 'k8stoken=<token>' -var 'access_key=<key>' -var 'secret_key=<secret>' -var 'key_name=keypair'`
```

Comment:
Resource aws_key_pair is actually imported from existing  keypair, terraform can't create them like aws UI does.
https://www.terraform.io/docs/providers/aws/r/key_pair.html

I think aws_key_pair can import existing keys (which answers questions above), it will store them under given name in aws.